mp/cell_mp.py

```diff
@@ -287,7 +287,7 @@ def get_msg_and_agg_func(self, adjacency):
         if adjacency == 'down':
             return self.message_and_aggregate_down
         elif adjacency == 'boundary':
-            return self.message_and_aggregate_boundaries
+            return self.message_and_aggregate_boundary
```

Comment:
I suppose this fixes a bug?

Reply:
Yes, found this "typo" while debugging some experiments. I could not really find any reference to `message_and_aggregate_boundar**ies**` in the code, and I assumed this had to be fixed this way. Let me know if you agree

Reply:
Yes. I thought we fixed this but it keeps re-emerging :D. Anyways, this relied on the default summing behaviour, which is what we use anyways everywhere.

Reply:
Exactly. I checked that as well. It seems we should not have problems with previous experimental results